addons/web/static/src/core/pager/pager.xml

```diff
@@ -11,7 +11,12 @@
                     <span class="o_pager_value d-inline-block border-bottom border-white" t-esc="value" t-on-click="onValueClick" />
                 </t>
                 <span> / </span>
-                <span class="o_pager_limit" t-esc="props.total" />
+                <t t-if="state.total == 10000">
```

Comment:
it's fine, but i would personally write it like this:
```xml
    <span class="o_pager_limit" t-on-click.stop="updateCounter">
     <t t-esc="state.total"/><t t-if="state.total===10000">+</t>
  </span>
```

Reply:
No, in one case there is a t-on-click and not in the other.